ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline.yml

```diff
@@ -273,9 +297,31 @@ spec:
           workspace: pipeline
           subPath: src
 
-    - name: dockerfile-creation
+    # check if cert_project already has package_lock
+    # and if the package_name match operator_name
+    - name: check-operator-name
```

Comment:
I know adding this to the CI pipeline was requested in the jira, but I wonder if it's actually necessary when the check is already in the hosted pipeline? Granted it'll help partners detect this specific problem early, but we don't duplicate all hosted pipeline validations in the CI pipeline. Maybe @Allda has a better idea on what the criteria on adding validations in the CI pipeline are.

Reply:
I don't think we should add this check to the CI pipeline, since we would now force the partner to run with a `full` set of tokens/keys to talk to Red Hat. If we put this check in we need to change all of our documentation (GitHub and Policy docs) to no longer allow a minimal run and are taking away the iterative approach that partners use the CI pipeline for.

Reply:
Decided not to add this check to CI pipeline - all changes are now only in hosted pipeline